### ISSUE TITLE
Remaining accessibility issues

### ISSUE THREAD

**Issue by alice** [Polymer/paper-checkbox#11]
- No focus style (I realise this is dependent on the Material Design spec)
- No keyboard event handling that I could figure out - see http://www.w3.org/TR/wai-aria-practices/#checkbox
